### ISSUE TITLE
[ui5-multi-combobox] missing focus outline

### ISSUE THREAD

**Issue by nnaydenow** [SAP/ui5-webcomponents#2263]
When the user navigates with TAB, the **ui5-multi-combobox** receives the focus, but does not show the focus outline around its input field.

**Expected:**
To show the focus outline when the component is accessed with the TAB key.

**Comment by MapTo0** [comment]
Fixed by https://github.com/SAP/ui5-webcomponents/pull/2431
